RELEASES.md

```diff
@@ -1,3 +1,105 @@
+Version 1.30.0 (2018-10-25)
+==========================
+
+Language
+--------
+- [This version marks the release of the 2018 edition of Rust.][54057]
+- [In the 2018 edition you can now use `crate` visibility keyword.][54057]
+  Equivalent to `pub(crate)`.
+- [You can now use keywords as identifiers using the raw identifiers
+  syntax (`r#`).][53236] e.g. `let r#struct = 2018;`
+- [Using anonymous parameters in traits is now deprecated with a warning and is
+  a hard error in the 2018 edition.][53272]
+- [You can now apply the `#[used]` attribute to static items to prevent the
+  compiler from optimising them away even if they appear to be unused.][51363]
+  e.g. `#[used] static FOO: u32 = 1;`
+- [You can now import and reexport macros using the `use` syntax.][50911]
```

Comment:
Proc macro definitions (`#[proc_macro]`/`#[proc_macro_attribute]`) was stabilized in 1.29 (https://github.com/rust-lang/rust/pull/52081), but never mentioned in release notes.
In 1.30 there's no reason to be secretive about it, so it can be announced.

Reply:
Tool attributes (`#[rustfmt::skip]`) are stabilized in 1.30 (https://github.com/rust-lang/rust/pull/53459).

Reply:
"macros" -> "macros from other crates"

\+ `#[macro_export]` macros are now placed into the root module of the crate that defines them (https://github.com/rust-lang/rust/pull/52234)
\+ `#[macro_export(local_inner_macros)]` is implemented (https://github.com/rust-lang/rust/pull/51496)

Reply:
To clarify there is no intent to be secretive about the feature. The first draft of the release notes is always just the PRs merged in the six weeks between the releases editorialised by me, with PRs tagged as `relnotes` always included.

Reply:
@petrochenkov Could you perhaps explain when you would use `#[macro_export(local_inner_macros)]` I understand the problem as laid out in the issue but I don't really know how to explain this to an end user of Rust.

Reply:
@Aaronepower A lot of crates were written with complex macros prior to this new system, assuming that `#[macro_export]` macros were always available to them. For example, Diesel has several macros that call inner helper macros, either for the purposes of factoring code, or for `#[cfg]`ing behavior. With the new system, a macro like `__diesel_parse_table!` wouldn't be available to `table!` unless it was imported by our users, or we put `$crate::` in front. The former would be ridiculous, and the latter is incompatible with Rust 2015 users. Hence, this attribute.